packages/react-i18n/src/tests/e2e/translations.test.tsx

```diff
@@ -57,4 +58,29 @@ describe('translations', () => {
 
     expect(component.text()).toBe(enTranslations.hello);
   });
+
+  it('uses custom interpolation', () => {
+    function WithI18nComponent() {
+      const [i18n] = useI18n({
+        id: 'MyComponent',
+        fallback: {hello: 'Hello {{name}} {{ surname }}!'},
+      });
+
+      return (
+        <div>{i18n.translate('hello', {name: 'foo', surname: 'bar'})}</div>
+      );
+    }
+
+    const manager = new I18nManager({
+      locale: 'en',
+      interpolate: MUSTACHE_INTERPOLATION,
```

Comment:
Usually, an app will use a single format across all its translations, so passing the option to the manager in the provider makes enough sense.